pkg/cache/cluster.go

```diff
@@ -134,8 +136,9 @@ func NewClusterCache(config *rest.Config, opts ...UpdateSettingsFunc) *clusterCa
 
 type clusterCache struct {
 	resyncTimeout time.Duration
-	syncTime      *time.Time
-	syncError     error
+
+	syncStatus *clusterCacheSync
```

Comment:
It does not need to be a pointer, does it?

Reply:
Done... I wanted to avoid accidental [copying of the mutex](https://eli.thegreenplace.net/2018/beware-of-copying-mutexes-in-go/), but it's true this way is cleaner.